src/mermaid.ts

```diff
@@ -81,10 +81,16 @@ export function mermaidify(
       (event.timing.startTime.getTime() - zeroMoment) / 1000;
 
     lines.push(
-      `${label} (${duration}s):${tag}, ${relativeStartTime}, ${duration}s`,
+      `${label} (${collapseSeconds(duration)}):${tag}, ${relativeStartTime}, ${duration}s`,
     );
   }
   lines.push("```");
 
   return lines.join("\n");
 }
+
+function collapseSeconds(duration: number): string {
```

Comment:
I think the `collapseSeconds` name is kinda odd. What do you think about renaming it like the below?
```suggestion
function formatDuration(duration: number): string {
```

Reply:
sure